assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
+    school = models.CharField(max_length=256)
+    school_class = models.CharField(max_length=128)
```

Comment:
School and school´s class need to change to models so these will be changed to foreigns. See the ticket TK-45.